### PR TITLE
Update doggos.md

### PR DESCRIPTION
**Pull request by ghost**
Co-authored-by:<ohjoycelau@github.com>

### CODE REVIEW

File: doggos.md [hunk 1]
```diff
@@ -1 +1,2 @@
 henry of henrythecoloradodog
+maya of mayapolarbear
```
Comment (ooyce) [inline doggos.md:2]: maya is the cutest

File: doggos.md [hunk 2]
```diff
@@ -1 +1,2 @@
 henry of henrythecoloradodog
```
Comment (ooyce) [inline doggos.md:1]: what about baloo?

File: doggos.md (hunk 1 above)
Comment (strawhat-123) [reply doggos.md:2]: she trained in puppy school for 2 months

Comment (strawhat-123) [reply doggos.md:2]: she loves watermelons

File: doggos.md (hunk 2 above)
Comment (ooyce) [reply doggos.md:1]: `henry and baloo of henrythecoloradodog`

**Review by ooyce — COMMENTED** [review]
diitto

File: doggos.md (hunk 1 above)
Comment (ooyce) [reply doggos.md:2]: omg that so much training 🙀

**Review by ooyce — APPROVED** [review]
yesp

**Review by strawhat-123 — COMMENTED** [review]
u should do this `henry is the dog in henrythecoloradodog`

**Review by strawhat-123 — CHANGES_REQUESTED** [review]
where is the comment i left?

File: doggos.md (hunk 2 above)
Comment (strawhat-123) [inline doggos.md:1]: `henry is the dog of henrythecoloradodog`

**Review by strawhat-123 — CHANGES_REQUESTED** [review]
please fix to identify henry as a dog

File: doggos.md (hunk 2 above)
Comment (strawhat-123) [inline doggos.md:1]: leave a comment

**Pull request closed without merging**